PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+
+# What you've done
+...
+
+# Why have you done it
+...
+
+# Screenshot (if visual)
+...
+
+# Any testing carried out
+...
```

Comment:
@gayanhewa 

//the automatic resolvers are available regardless of the template//

I didn't exactly get this. How does a PR link to an issue without an issue ID?